css/pages/coc.css

```diff
@@ -4,6 +4,10 @@
   font-weight: 550;
   padding: 20px;
  }
+
+ body {
+  font-family: 'Roboto', sans-serif;
+}
```

Comment:
Doesn't this come from `main.css`?

Reply:
Yeah ankush, I haven't seen that. I just had to add link in html pages. Sorry my bad. Will remove it in next pr.